src/StartMenu.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import logo from './logo.png';
+
+function StartMenu(props) {
+  const [cacoSpin, setCacoSpin] = useState(false);
+  return (
+      <header className="App-header">
+        <img
+          src={logo}
+          onMouseEnter={() => setCacoSpin(true)}
+          onMouseLeave={() => setCacoSpin(false)}
+          className={ cacoSpin ? "App-logo-reversed" : "App-logo" }
+          alt="logo"
+        />
+          <h1 className="Main-text">Welcome to the hell, marine!</h1>
+  <button className="Button Button-start" onClick={() => {props.setButtonStartQuiz(true)}}>I'm too young to die!</button>
```

Comment:
Correct the indentation here plz.

Reply:
Don't understand correctly what is wrong here. ClassNames?

Reply:
http://mrbool.com/importance-of-code-indentation/29079

Reply:
I think fixed.